.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,38 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Shouldn't these files be removed from the PR now that `.idea` is in the .gitignore?